src/Metrics.ts

```diff
@@ -18,21 +18,22 @@ export class Metrics {
     public readonly notificationsServiceUp = new Gauge({ name: "hookshot_notifications_service_up", help: "Is the notification service up or down", labelNames: ["service"], registers: [this.registry]});
     public readonly notificationsWatchers = new Gauge({ name: "hookshot_notifications_watchers", help: "Number of notifications watchers running", labelNames: ["service"], registers: [this.registry]});
 
-    private readonly matrixApiCalls = new Counter({ name: "matrix_api_calls", help: "The number of Matrix client API calls made", labelNames: ["method"], registers: [this.registry]});
-    private readonly matrixApiCallsFailed = new Counter({ name: "matrix_api_calls_failed", help: "The number of Matrix client API calls which failed", labelNames: ["method"], registers: [this.registry]});
+    private readonly matrixApiCalls = new Counter({ name: "hookshot_matrix_api_calls", help: "The number of Matrix client API calls made", labelNames: ["method"], registers: [this.registry]});
```

Comment:
I wasn't sure about this one since we use it in other bridges, and it might break compat with other dashes (similar to the node exporter default metrics)

Reply:
Yep, that's the idea. Anything common between bridges should be the same so people can reuse dashes (there is a "job" parameter provided by prometheus that allows you to distinguish if you need to)